build-tools/package.json

```diff
@@ -39,6 +39,8 @@
     "policy-check": "node packages/build-cli/bin/dev check policy --exclusions packages/build-tools/data/exclusions.json",
     "policy-check:asserts": "node packages/build-cli/bin/dev check policy --exclusions packages/build-tools/data/exclusions.json --handler assert-short-codes --fix",
     "policy-check:fix": "node packages/build-cli/bin/dev check policy --exclusions packages/build-tools/data/exclusions.json --excludeHandler assert-short-codes --fix",
+    "prettier": "lerna run prettier --parallel --no-sort --stream",
```

Comment:
Is this expected script for both `prettier` and `prettier:fix`? Currently, `package.json` in `root` has `lerna run format` for `format` script and does not include `prettier` and `prettier:fix` inside the file. 

This is the case for `azure/package.json`, `build-tools/package.json`, 'server/package.json`, `server/histroian/package.json`, & `server/routerlicious/package.json`.

The current policy handler does not add `prettier` and `prettier:fix` to the `package.json` if the `format` script includes `lerna run` (in other words the package.json is in `root`).

It might be good idea to clarify when we would like to add `prettier` and `prettier:fix` scripts to the `package.json`.

Reply:
Besides this, everything looks good to me 😄 !

Reply:
Thanks for pointing this out! I thought I was being consistent but was doing the opposite. :)